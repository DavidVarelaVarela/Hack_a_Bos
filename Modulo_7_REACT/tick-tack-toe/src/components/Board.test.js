import React from 'react';
import { Board } from './Board';
import { render } from '@testing-library/react';

test('<Board />', () => {
  const { container } = render(<Board />);
  expect(container).toMatchSnapshot();
});
