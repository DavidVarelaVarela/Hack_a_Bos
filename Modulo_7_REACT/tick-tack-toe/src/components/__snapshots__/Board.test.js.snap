// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Board /> 1`] = `
<div>
  <div
    class="status"
    data-testid="status"
  >
    Next player: undefined (X)
  </div>
  <div
    class="grid"
  >
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
    <button
      class="square"
      data-testid="square"
    />
  </div>
  <div
    class="buttons"
  >
    <button
      class="btn"
    >
      Reset
    </button>
    <button
      class="btn"
    >
      Enter Players
    </button>
  </div>
</div>
`;
